src/assets/icons/agreement-rtl.svg

```diff
@@ -1 +1 @@
+<svg width="240" height="204" style="--icon-color: transparent;" xmlns="http://www.w3.org/2000/svg"><circle r="102" transform="matrix(-1 0 0 1 120 102)" fill="var(--agreement-background, #E6F9F2)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M74 64l26-26h60.002a6 6 0 016 6v114a8 8 0 01-8 8H80a6 6 0 01-6-6V64z" fill="var(--agreement-form, #fff)" stroke="var(--agreement-form-outline, #333D48)" stroke-width="2" stroke-linejoin="round"/><path fill-rule="evenodd" clip-rule="evenodd" d="M143.376 39c-.247 1.63-.376 3.3-.376 5 0 14.368 9.183 26.592 22 31.122V39h-21.624z" fill="var(--agreement-form-shadow, #D6DDE3)"/><circle r="32" transform="matrix(-1 0 0 1 178 42)" fill="var(--agreement-padlock-background, #fff)" stroke="var(--agreement-padlock-background-outline, #333D48)" stroke-width="2"/><path fill-rule="evenodd" clip-rule="evenodd" d="M178 24a9 9 0 019 9v8a9 9 0 01-9 9 9 9 0 01-9-9v-8a9 9 0 019-9zm0 26a5 5 0 005-5V34a5 5 0 00-10 0v11a5 5 0 005 5z" fill="var(--agreement-padlock-hasp, #E7EBEE)"/><path d="M186 33a8 8 0 00-8-8v-2c5.523 0 10 4.477 10 10h-2zm0 8v-8h2v8h-2zm-8 8a8 8 0 008-8h2c0 5.523-4.477 10-10 10v-2zm-8-8a8 8 0 008 8v2c-5.523 0-10-4.477-10-10h2zm0-8v8h-2v-8h2zm8-8a8 8 0 00-8 8h-2c0-5.523 4.477-10 10-10v2zm0 24a4 4 0 004-4h2a6 6 0 01-6 6v-2zm4-4V34h2v11h-2zm0-11a4 4 0 00-4-4v-2a6 6 0 016 6h-2zm-4-4a4 4 0 00-4 4h-2a6 6 0 016-6v2zm-4 4v11h-2V34h2zm0 11a4 4 0 004 4v2a6 6 0 01-6-6h2z" fill="var(--agreement-padlock-hasp-outline, #333D48)"/><rect width="24" height="24" rx="12" transform="matrix(-1 0 0 1 190 36)" fill="var(--agreement-padlock, #FFC037)" stroke="var(--agreement-padlock-outline, #EBA100)" stroke-width="2"/><circle r="3" transform="matrix(-1 0 0 1 178 46)" fill="var(--agreement-padlock-lock, #333D48)"/><rect width="2" height="8" rx="1" transform="matrix(-1 0 0 1 179 46)" fill="var(--agreement-padlock-lock, #333D48)"/><circle r="32" transform="matrix(-1 0 0 1 62 158)" fill="var(--agreement-tick-background, #fff)" stroke="var(--agreement-tick-background-outline, #333D48)" stroke-width="2"/><path fill-rule="evenodd" clip-rule="evenodd" d="M46 161l4-4 6 6 16-16 4 4-20 20-10-10z" fill="var(--agreement-tick, #2DC692)" stroke="var(--agreement-tick-outline, #1AA175)" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path fill-rule="evenodd" clip-rule="evenodd" d="M100 41v18a8 8 0 01-8 8H75v-2l25-24z" fill="var(--agreement-form-shadow, #D6DDE3)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M100 38v18a8 8 0 01-8 8H74l26-26z" fill="var(--agreement-form-corner, #C1CBD5)" stroke="var(--agreement-form-outline, #333D48)" stroke-width="2" stroke-linejoin="round"/><path d="M146 89h-40M146 81H94M146 97H98M146 105h-40M146 113H94M146 121h-44M146 135h-24" stroke="var(--agreement-form-text, #C1CBD5)" stroke-width="2"/></svg>
```

Comment:
@ArkadiuszKrzymienski, please move the element per line. IMO SVG loader should optimize this file. For us element per line should be more readable <3 :thank_you:

Reply:
:thank_you: